### PR TITLE
Add video and empty portfolio key - Noah Platton

### PR DESCRIPTION
**Pull request by nplatton**

### CODE REVIEW

File: lytical/roster.json
Comment (rom-30) [inline lytical/roster.json:71]: Can't approve the pr, if there is no link to your portfolio :(

**Review by rom-30 — CHANGES_REQUESTED** [review]

File: lytical/roster.json
Comment (nplatton) [reply lytical/roster.json]: Oh oops! I've added it now!
...
defo not still working at this time...

**Review by rom-30 — APPROVED** [review]

**Pull request merged**
